torch/quantization/observer.py

```diff
@@ -126,6 +168,8 @@ def observer(observer_cls, **kwargs):
     return partial(observer_cls, **kwargs)
 
 def default_observer(**kwargs):
+    # Restrict activations to be in the range (0,127)
+    kwargs.setdefault("reduce_range", True)
```

Comment:
Should it be False by default?

Reply:
I think it should be true so that when someone uses the default observer, they get the best possible accuracy. Setting reduce_range to be true does that.

Reply:
Hmm, I just thought reducing range is not the normal case, since it reduce the number of levels.

Reply:
We need the default to work without any changes for x86 quantization and provide the best performance. Leaving this as an option to the user runs the risk of having poorer accuracy for quantized models.